src/materials/MeshPhongMaterial.js

```diff
@@ -177,6 +177,23 @@ THREE.MeshPhongMaterial.prototype.toJSON = function () {
 	if ( this.blending !== THREE.NormalBlending ) data.blending = this.blending;
 	if ( this.side !== THREE.FrontSide ) data.side = this.side;
 
+	if(this.map !== null && this.map !== THREE.Texture ) data.map = this.map.uuid;
```

Comment:
Right-o -- was wondering if you were on that but it was still in dev so wasn't sure :p .. goodie good :)